packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
```

Comment:
I would say all used style props are probably themeable, my list would include:
border, background, box-shadow, padding (based on relative "sizing set" of css-variables for entire theme)